src/store/modules/events/index.ts

```diff
@@ -450,6 +471,23 @@ const module: Module<EventsModuleState, RootState> = {
       state.repetitions[key] = repetition;
     },
 
+    /**
+     * Mark event as visited in state
+     *
+     * @param {EventsModuleState} state
+     * @param {string} projectId - project event is related to
+     * @param {string} eventId - visited event
+     */
+    [MutationTypes.MARK_AS_VISITED](state, { projectId, eventId }) {
+      const userId = store.state.user.data.id;
```

Comment:
what is `store`?

Reply:
VueX store

Reply:
better to call `this.store` https://github.com/vuejs/vuex/issues/344

Reply:
or pass userId from the action. https://github.com/vuejs/vuex/issues/344#issuecomment-249880816